umap/static/umap/test/Util.js

```diff
@@ -1,397 +1,4 @@
 describe('L.Util', function () {
```

Comment:
I'm surprised to see all those tests removed here ? Should this have been done in that other PR moving utils to a module ? Or is this just missing a rebase ?

Reply:
yep, it's broken in the current master, and fixed here! 🙈